app.py

```diff
@@ -197,50 +193,101 @@ def _schedule_core_solver_jobs(index_urls: List[str], result_api: str) -> None:
     help="AICoE URL base for discovering packages.",
 )
 @click.option(
-    "--result-api",
-    "-r",
-    type=str,
-    envvar="THOTH_RESULT_API_URL",
-    help="AICoE URL base for discovering packages.",
+    "--initialize-schema",
+    required=False,
+    is_flag=True,
+    envvar="THOTH_INIT_JOB_INITIALIZE_SCHEMA",
+    help="Initialize schema in Thoth Knowledge Graph.",
+)
+@click.option(
+    "--register-indexes",
+    required=False,
+    is_flag=True,
+    envvar="THOTH_INIT_JOB_REGISTER_INDEXES",
+    help="Register indexes in Thoth Knowledge Graph.",
 )
 @click.option(
-    "--register-indexes-only",
+    "--solve-core-packages",
     required=False,
     is_flag=True,
-    envvar="THOTH_INIT_JOB_REGISTER_INDEXES_ONLY",
-    help="Do not schedule solver jobs, only register indexes.",
+    envvar="THOTH_INIT_JOB_REGISTER_CORE_PACKAGES",
+    help="Schedule solver jobs for core packages.",
 )
-def cli(verbose: bool = False, dry_run: bool = False, result_api: str = None, index_base_url:
-        str = None, register_indexes_only: bool = False):
+@click.option(
+    "--solve-data_science-packages",
+    required=False,
+    is_flag=True,
+    envvar="THOTH_INIT_JOB_REGISTER_DATA_SCIENCE_PACKAGES",
+    help="Schedule solver jobs for data science packages.",
+)
+def cli(
+    verbose: bool = False,
+    dry_run: bool = False,
+    index_base_url: str = None,
+    initialize_schema: bool = False,
+    register_indexes: bool = False,
+    solve_core_packages: bool = False,
+    solve_data_science_packages: bool = False,
+):
     """Register AICoE indexes in Thoth's database."""
     graph = None
 
+    if not dry_run:
+        graph = GraphDatabase()
+        graph.connect()
+
     if verbose:
         _LOGGER.setLevel(logging.DEBUG)
 
     if not index_base_url.endswith("/"):
         index_base_url += "/"
 
-    if not dry_run:
-        graph = GraphDatabase()
-        graph.connect()
-        _LOGGER.info("Initializing schema")
-        graph.initialize_schema()
-    elif dry_run:
-        _LOGGER.info("dry-run: not talking to OpenShift or Dgraph...")
+    if initialize_schema:
+        if not dry_run:
+            _LOGGER.info("Initializing schema")
+            graph.initialize_schema()
+        elif dry_run:
+            _LOGGER.info("dry-run: not talking to Thoth Knowledge Graph...")
```

Comment:
It might be good idea to report what would be done if dry-run would not be passed.